src/pages/QrCode/QrCode.tsx

```diff
@@ -0,0 +1,143 @@
+import { Box, Button, Paper, Typography } from '@mui/material';
+import { TitleBox } from '@pagopa/selfcare-common-frontend';
+import { useState, useEffect } from 'react';
+import { QRCodeSVG } from 'qrcode.react';
+import { TransactionCreationRequest } from '../../api/generated/payment-qr-code/TransactionCreationRequest';
+import MerchantDataForm from '../../components/MerchantDataForm';
+import TransactionDataForm from '../../components/TransactionDataForm';
+import { createTransaction, getTransaction } from '../../services/PaymentQrCodeApiService';
+import { TransactionResponse } from '../../api/generated/payment-qr-code/TransactionResponse';
+
+const initiatlState = {
+  amountCents: 0,
+  idTrxIssuer: new Date().getTime().toString(),
+  initiativeId: '63d26bbc0e71e44bb08de293',
```

Comment:
let's read the default value from ENV using  utils.env module

Put the actual value inside the deployment pipeline for DEV env
in UAT env use the "INITIATIVEID" with a TODO to remember to fill it later